packages/wallet/backend/tests/account/controller.test.ts

```diff
@@ -0,0 +1,221 @@
+import { createContainer } from '@/createContainer'
+import { Bindings } from '@/app'
+import { env } from '@/config/env'
+import { Container } from '@/shared/container'
+import { createApp, TestApp } from '@/tests/app'
+import { Knex } from 'knex'
+import { Request, Response } from 'express'
+import {
+  createRequest,
+  createResponse,
+  MockRequest,
+  MockResponse
+} from 'node-mocks-http'
+import type { AuthService } from '@/auth/service'
+import { applyMiddleware } from '@/tests/utils'
+import { withSession } from '@/middleware/withSession'
+import type { UserService } from '@/user/service'
+import { mockedListAssets, mockLogInRequest } from '../mocks'
+import { AccountController } from '@/account/controller'
+import { AccountService } from '@/account/service'
+import { User } from '@/user/model'
+import { Account } from '@/account/model'
+import { faker } from '@faker-js/faker'
+import { BaseError } from '@/errors/Base'
+
+describe('Asset Controller', (): void => {
+  let bindings: Container<Bindings>
+  let appContainer: TestApp
+  let knex: Knex
+  let authService: AuthService
+  let userService: UserService
+  let accountController: AccountController
+  let accountService: AccountService
+  let req: MockRequest<Request>
+  let res: MockResponse<Response>
+  const mockedAsset = mockedListAssets[0]
+  const accountName = 'Hmmpotato'
```

Comment:
:D This is a funny account name :D, but maybe maybe we should use like official names? 'My account', 'Test account' :D 'Test potato' :D